common/src/test/java/com/microsoft/identity/common/CacheKeyValueDelegateTest.java

```diff
@@ -95,6 +95,26 @@ public void accessTokenCreateCacheKeyComplete() {
         assertEquals(expectedKey, mDelegate.generateCacheKey(accessToken));
     }
 
+    @Test
+    public void accessTokenCreateCacheKeyCompleteWithEscapeSequences() {
+        final AccessTokenRecord accessToken = new AccessTokenRecord();
+        accessToken.setHomeAccountId(HOME_ACCOUNT_ID);
+        accessToken.setEnvironment("\r\f\n\t" + ENVIRONMENT + "\r\f\n\t");
```

Comment:
Is this the only field that has this behavior?

Reply:
They all should - can broaden this test to make this clearer.